.github/ISSUE_TEMPLATE/2-BUG.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Problem with the pub client
+about: The pub client is printing a stack trace or failing unexpectedly.
+
+---
+
+# Environment
+
+- `pub version` or `flutter packages pub version`:
+- OS version:
+- Are there any firewalls or proxies that could be impacting the problem?
+
+# Problem
+
+<!--
+  Please describe the problem in detail, including the specific command that is
+  failing.
+-->
```

Comment:
We're often wondering what packages users were trying to fetch, or what they were trying to publish, maybe we could ask them to attach the `pubspec.yaml`.

        If relevant, what does the `pubspec.yaml` file for your project look like:
        ```yaml
        <copy/paste pubspec.yaml>
        ```

Reply:
Done